pkgs/tools/security/rnp/default.nix

```diff
@@ -34,7 +35,7 @@ stdenv.mkDerivation rec {
     "-DDOWNLOAD_RUBYRNP=off"
   ];
 
-  nativeBuildInputs = [ asciidoctor cmake gnupg gtest pkg-config python3 ];
+  nativeBuildInputs = [ asciidoctor cmake gnupg gtest pkg-config python3 git ];
```

Comment:
Why is git needed? We should probably patch it out and set the version numbers manually.

Reply:
This is an issue that was introduced recently.  [There's a ticket tracking it.](https://github.com/rnpgp/rnp/issues/1579)  We hope to have it taken out by next version.

Maybe we can add a patch, as you suggest, so it can build successfully without git.

Reply:
Issue with git is fixed via https://github.com/rnpgp/cmake-versioning/pull/7 (and will land RNP repository soon as well).

Reply:
Thanks @ni4! 

Also, I've now included a patch here in the latest update for this particular RNP version so git is not needed any more.